ros/lib/consume.py

```diff
@@ -0,0 +1,16 @@
+from confluent_kafka import Consumer
+from ros.lib.config import INSIGHTS_KAFKA_ADDRESS, GROUP_ID, kafka_auth_config
+
+
+def init_consumer(kafka_topic):
```

Comment:
@r14chandra  - could you please create a separate PR for this refactoring with consume & produce file?